File: output/qwen_2.5_72b_instruct/avatar/direct_translation/java/python/codeforces_621_A.py

import sys

input = sys.stdin.read().split()
n = int(input[0])
list = [int(x) for x in input[1:]]
odd = []
sum = 0
for i in list:
    if i % 2 == 0:
        sum += i
    else:
        odd.append(i)
odd.sort()
for i in odd:
    sum += i
if len(odd) % 2 != 0:
    sum -= odd[0]
print(sum)

File: output/gpt_4o/avatar/direct_translation/python/java/codeforces_621_A.java

import java.util.*;

public class codeforces_621_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        List<Integer> lst = new ArrayList<>();
        String[] input = scanner.nextLine().split(" ");
        for (int i = 0; i < n; i++) {
            lst.add(Integer.parseInt(input[i]));
        }
        Collections.sort(lst, Collections.reverseOrder());
        int od = 0;
        for (int i : lst) {
            if ((i & 1) == 1) {
                od += 1;
            }
        }
        int sum = 0;
        boolean ok = false;
        for (int i = 0; i < n; i++) {
            if (lst.get(i) % 2 == 0) {
                sum += lst.get(i);
            } else {
                if (ok || od > 1) {
                    sum += lst.get(i);
                    od -= 1;
                    ok = !ok;
                }
            }
        }
        System.out.println(sum);
    }
}

